tests/functional/api/test_topics.py

```diff
@@ -4,11 +4,18 @@
 """
 
 
-def test_topics(gl):
+def test_topics(gl, gitlab_version):
     assert not gl.topics.list()
 
-    topic = gl.topics.create({"name": "my-topic", "description": "My Topic"})
+    create_dict = {"name": "my-topic", "description": "My Topic"}
+    if gitlab_version.major >= 15:
+        create_dict["title"] = "my topic title"
+    topic = gl.topics.create(
+        {"name": "my-topic", "title": "my topic title", "description": "My Topic"}
+    )
     assert topic.name == "my-topic"
+    if gitlab_version.major >= 15:
+        assert topic.title == "my topic title"
```

Comment:
FWIW I don't think we need to test multiple major versions as tests get complicated quickly, but I agree it'll be helpful having version checks when developing new features.